site/_scss/blocks/_post-banner.scss

```diff
@@ -0,0 +1,13 @@
+.post-banner {
+  --link-color: var(--color-red-darkest);
+  --link-visited-color: var(--color-red-darkest);
+  --link-rgb-background: var(--rgb-red-darkest);
+  background-color: get-color('red-50');
```

Comment:
Isn't that `var(--color-red-lightest)` as in the warning aside?

Reply:
`var(--color-red-lightest)` in the warning aside is using 'red-50' var with 0.5 opacity. I asked Dan to confirm about this and he said this announcement banner is floating banner so it makes sense to have it solid.

Reply:
Generally, we should be using colors from _theme.scss, because if you use colord directly you cannot implement theme in a correct way (e.g. imagine you need to introduce dark theme in next sprint, which is actually very realistic).

I don't see why it wouldn't be --color-red-lightest or --color-red-lighter.

Reply:
The existing code is:
```
  --color-red-lightest: #{rgb(get-color('red-50'), 0.4)};
  --color-red-lighter: #{rgb(get-color('red-50'), 0.5)};
```

However, Dan wants the new announcement banner to have the solid colour (without opacity). So I cannot use `--color-red-lightest` or `--color-red-lighter`. Is that okay if I add a new variable to the theme,  like `--color-red-solidLighter: #{get-color('red-50')};`?

Reply:
Yep, then add `--color-red-lightest-solid` to theme.scss

Reply:
Dan have confirmed me last week that we can use `color-red-lightest` (with opacity) for the outdated banner.